internal/common/metrics/capturing_stats_reporter.go

```diff
@@ -38,7 +38,7 @@ import (
 func NewMetricsScope(isReplay *bool) (tally.Scope, io.Closer, *CapturingStatsReporter) {
 	reporter := &CapturingStatsReporter{}
 	opts := tally.ScopeOptions{Reporter: reporter}
-	scope, closer := tally.NewRootScope(opts, time.Second)
+	scope, closer := tally.NewRootScope(opts, 100*time.Millisecond)
```

Comment:
Is this changing reporting / aggregation frequency or something? 10x is a lot, just want to make sure this won't mean we're overloading things

Reply:
This is only for a test utility. From a few lines above:

> // This file contains test helpers only. They are not private because they are used by other tests.

Having it here with the rest of the code is a bit confusing, but :shrug:. Maybe I'll take the advice from https://github.com/temporalio/sdk-go/pull/635#discussion_r751457854 and move it off to another package if I'm allowed to do that refactor.

Reply:
Ah, got it. Makes sense.